src/AzSK.ADO/Framework/Helpers/CommonHelper.ps1

```diff
@@ -0,0 +1,64 @@
+using namespace Newtonsoft.Json
```

Comment:
we already have Helper.ps1 for storing generic helper functions. We should keep it there.